examples/slab2.py

```diff
@@ -13,19 +13,48 @@
 import cmocean
 import cartopy.crs as ccrs
 
-# Load a geometry of the Alaska subduction zone
-grid = rh.fetch_slab2(zone="south_america")
-print(grid)
-
-# Plot the sibductin zones using cartopy
 plt.figure(figsize=(10, 5))
 ax = plt.axes(projection=ccrs.Robinson())
-pc = grid.depth.plot.pcolormesh(
-    cmap=cmocean.cm.thermal_r,
-    cbar_kwargs=dict(pad=0.01, aspect=30),
-    ax=ax,
-    transform=ccrs.PlateCarree(),
-)
+
+zones = {
+    "alaska",
+    "calabria",
+    "caribbean",
+    "cascadia",
+    "central_america",
+    "cotabalo",
+    "halmahera",
+    "hellenic",
+    "himalaya",
+    "hindu_kush",
+    "izu_bonin",
+    "kamchatka",
+    "kermadec",
+    "makran",
+    "manila_trench",
+    "muertos_trough",
+    "new_guinea",
+    "pamir",
+    "philippines",
+    "puysegur",
+    "ryukyu",
+    "scotia_sea",
+    "solomon_islands",
+    "south_america",
+    "sulawesi",
+    "sumatra_java",
+    "vanuatu",
+}
```

Comment:
There's no need to rewrite every zone available on Slab2.
You can import the `ZONES` dictionary directly and then iterate over its keys:
```python
from rockhound.slab2 import ZONES

...

for zone in ZONES:
    grid = rh.fetch_slab2(zone)
    ...
```

Reply:
Done!